src/message/kafka/headers.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+import { CloudEvent, CONSTANTS, Headers } from "../..";
+
+/**
+ * The set of CloudEvent headers that may exist on a Kafka message
+ */
+export const KAFKA_CE_HEADERS: { [key: string]: string } = Object.freeze({
```

Comment:
I believe the type could be stronger or be omitted, specifically the `key: string` type.

```ts
Readonly<{
    ID: string;
    TYPE: string;
    SOURCE: string;
    SPEC_VERSION: string;
    TIME: string;
    SUBJECT: string;
    DATACONTENTTYPE: string;
    DATASCHEMA: string;
}>
```

Otherwise, I'm not getting autocompletion for `KAFKA_CE_HEADERS.`...